source/install/linuxone.html.md

```diff
@@ -0,0 +1,221 @@
+* RHEL (8.0)
+
+# General Notes:
+
+* Please make sure the system hardware model you will deploy on is at least IBM z13 or above, lower than this model is not supported. 
+* If no explicit state, the steps need to be performed on all relevant nodes.
+* To simplify this doc, we use root user to create this demo, but a normal user to perform the deploy may be required in certain case and it’s ok, in that case you will need create a new OS account and configure sudo for the system.
+
+# Prerequisites
+
+Using Packstack, you can perform single-node/multi-node deployment by customizing the answer file for installation. In this guide, we will demonstrate how to perform a single-node deployment (that is one node to provide the controller service + compute service + network service, etc).
+
+## Software
+Red Hat Enterprise Linux (RHEL) 8 is the only verified version at the creation time of this how-to, we will update this guide if we verify other releases and Linux distributions. 
+
+## Hardware 
+Here is the minimum recommended configuration for an LPAR of IBM Z or IBM LinuxONE system. It should have at least 16GB RAM, 2 logical IFLs configured with hardware virtualization extensions, and two network adapters.
+
+| Server | CPU | Memory | Storage | Network Adapter |
+| ------ | --- | ------ | ------- | --------------- |
+| Physical Servers| 2 logical IFLs | >= 16GB | 50 GB | 1 x Admin NIC (can be shared OSA port) 1 x Data NIC (dedicate OSA port) | 
+
+*If you perform a single-node deployment, please allocate more memory on the host to gain a stable deployment. 
+
+## Install Operating System on an LPAR
+
+Install a base operating system on the LPAR, minimum installation is OK.
+ 
+Follow below RHEL8 official installation guide, if you are not familiar with the installation process. 
+https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/performing_a_standard_rhel_installation/installing_red_hat_enterprise_linux_on_ibm_z
+
+## Disable NetworkManager
+
+For a working and stable deployment, below services are recommended to be disabled and stopped, but with the traditional network service enabled and up.
+
+Follow this guide for the details:
+* https://access.redhat.com/solutions/4365931 
+
+Issue below commands on the terminal of every participated node.
+
+```Shell
+systemctl disable firewalld.service
+systemctl stop firewalld.service
+systemctl disable NetworkManager.service
+systemctl stop NetworkManager.service
+systemctl enable network.service
+systemctl start network.service
+```
+
+## Make sure your IP and hostname were listed in /etc/hosts.
+```Shell
+cat /etc/hosts
+
+172.16.36.194   	lpar94
+```
+# Software repositories
+## Enable YUM repos
+You need enable Redhat subscription to get update packages from Red Hat, and add following repos to get OpenStack packages and OpenStack dependency packages.
+
+```Shell
+[rdo-train-upstream]
+name=rdo-train-upstream
+baseurl=https://trunk.rdoproject.org/rhel8-train/current-tripleo/
```

Comment:
Mid-term we'll probably get rid of this builder, could you use https://trunk.rdoproject.org/centos8-train/puppet-passed-ci/ instead?

Reply:
I will update to use the correct one. Thanks.